package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "bootstrap": "lerna bootstrap --ci --reject-cycles",
     "build": "lerna run build --stream",
+    "build:watch": "lerna run build:watch --stream",
```

Comment:
I don't think this works for watching all of the repos since lerna is going to wait for a package's dependencies in the build graph to complete before running their dependents but none of the `build:watch` scripts ever actually completes on its own. For example,`webcomponentsjs` will never be rebuilt as a result of changes to one of the individual polyfills.

Reply:
Ah, good point, I hadn't tested webcomponentsjs (there aren't any other inter package dependencies I don't think, so it didn't really matter for those).

Switched from `--stream` to `--parallel`. The latter ignores dependencies (and they actually use a "watch" command as the example!): https://github.com/lerna/lerna/tree/master/commands/run#--parallel

Also extended the chokidar watched paths for webcomponentsjs to include the JS files of its dependencies.

Reply:
Ok, this seems to be building everything in the graph for me now. I think we should get the tests package included in there too though. Particularly, starting from a clean checkout, `npm i && npm run bootstrap && npm run build:watch` in one shell and `npm run test` in another should pass. I haven't investigated much of what to do to fix it but I tried running `npx lerna run build --scope=tests` right after all the other builds settled and tried again but got the same error.

Reply:
Ah good catch. I didn't realize there was anything to build in the tests package. Fixed.

Tests are all passing for me locally.

Reply:
I tried again and wasn't seeing them passing so I did a bit more digging and the custom elements polyfill didn't get built. I wasn't sure what was happening exactly so I removed concurrently and changed chokidar to watch the TS inputs and run `npm run build` instead and that seemed to fix it. I think there's some weird edge case where chokidar isn't recognizing that files were written by tsc.

Reply:
Interesting, maybe a linux vs mac thing? (I'm on linux). Removed concurrently for now.